wger/exercises/models.py

```diff
@@ -171,7 +171,8 @@ class Exercise(AbstractSubmissionModel, AbstractLicenseModel, models.Model):
         verbose_name=_('Category'), on_delete=models.CASCADE)
     description = models.TextField(max_length=2000,
                                    verbose_name=_('Description'),
-                                   validators=[MinLengthValidator(40)])
+                                   default='Keeps the body fit',
+                                   validators=[MinLengthValidator(10)])
```

Comment:
Why the reduction in the limit of MinLengthValidator? Isn't 10 a bit low for the exercise description?

Reply:
This was to allow the addition of a default description just in case a user adds an exercise without giving a description.

Reply:
Okay.